src/screens/PinCodeUnlock/PinCodeUnlock.js

```diff
@@ -122,10 +121,10 @@ class PinCodeUnlock extends React.Component<Props, State> {
   };
 
   showBiometricLogin() {
-    const { loginWithPrivateKey, connectionKeyPairs: { data: connKeys, lastConnectionKeyIndex } } = this.props;
+    const { loginWithPrivateKey } = this.props;
     const { biometricsShown, supportedBiometryType } = this.state;
 
-    if (biometricsShown || connKeys.length <= PRE_KEY_THRESHOLD || lastConnectionKeyIndex === -1) {
+    if (biometricsShown) {
       Toast.show({
```

Comment:
Not sure if we need this toast here

Reply:
Not sure as well :D But since we're going to change the auth flow in the next PR, I think we could leave this